hit-base-web/src/main/webapp/views/profileverification.html

```diff
@@ -0,0 +1,78 @@
+<style id="antiClickjack">
```

Comment:
Isn't this specific to GVT ? Not sure we want it here. Any thought ?

Reply:
yes correct will remove tomorrow